src/coreclr/jit/emitriscv64.cpp

```diff
@@ -3528,6 +3472,22 @@ bool emitter::emitDispBranch(
 void emitter::emitDispIllegalInstruction(code_t instructionCode)
 {
     printf("RISCV64 illegal instruction: 0x%08X\n", instructionCode);
+    assert(!"RISCV64 illegal instruction");
+}
+
+void emitter::emitDispImmediate(ssize_t imm, bool newLine /*= true*/, unsigned regBase /*= REG_ZERO*/)
+{
+    if (emitComp->opts.disDiffable && (regBase != REG_FP) && (regBase != REG_SP))
+    {
+        printf("0xd1ffab1e");
```

Comment:
nit: in other places hex is upper case
```suggestion
        printf("0xD1FFAB1E");
```
(IIRC, generally upper cased hex digits are preferred in native and managed code)